cms/constants.py

```diff
@@ -16,5 +16,8 @@
     "images/webinars/webinar-default-004.jpg",
     "images/webinars/webinar-default-005.jpg",
 ]
+WEBINAR_HEADER_BANNER = "images/webinars/webinar-header-banner.jpg"
```

Comment:
nit: Let's add a comment above these constants to distinguish them better. This is a kind of take-or-leave thing.

```# ********** CONSTANTS FOR WEBINARS**************```